src/Sylius/Bundle/ResourceBundle/Form/DataTransformer/ResourceToIdentifierTransformer.php

```diff
@@ -49,7 +49,7 @@ public function __construct(RepositoryInterface $repository, $identifier = 'id')
     public function transform($value)
     {
         if (null === $value) {
-            return '';
+            return null;
```

Comment:
Are you sure about that change? I recall that according to [symfony docs](http://symfony.com/doc/current/form/data_transformers.html#creating-the-transformer) we should return empty string here.